### ISSUE TITLE
Scripts executed from onClick handler

### ISSUE THREAD

**Issue by firda-cze** [KSP-KOS/KOS#2280]
I am opening separate issue for this. It relates to #2239 and #2277.

Callbacks are triggers, triggers cannot interrupt other triggers, thus running script with triggers (e.g. `lock throttle` which is also a trigger) from callbacks (`onClick`) does not work (`lock throttle` never executed, blocked by the callback the `run` is inside).
